public/preload.js

```diff
@@ -16,6 +16,26 @@ Sentry.init({
 })
 
 process.once('loaded', () => {
+  const oldAccessDataRaw = localStorage.getItem('wallet:accessData');
+  if (oldAccessDataRaw) {
+    // When migrating old wallets initialized with hardware wallets we need to clean the localStorage
+    const accessData = JSON.parse(oldAccessDataRaw);
+    const isHardware = localStorage.getItem('wallet:type') === 'hardware';
+    if (accessData.from_xpub || isHardware) {
+      const uniqueId = localStorage.getItem('app:uniqueId');
+      const backup = localStorage.getItem('wallet:backup');
+      const network = localStorage.getItem('wallet:network');
+      const server = localStorage.getItem('wallet:server');
+      localStorage.clear();
+      localStorage.setItem('localstorage:started', 'true');
+      localStorage.setItem('localstorage:backup', backup);
+      localStorage.setItem('localstorage:network', network);
+      localStorage.setItem('localstorage:server', server);
```

Comment:
question: Based on our experience with the recent #537 and #547 bugfixes, should we also update the wallet lib's `server` and `networkName` configurations here?

Reply:
This has since been removed, the only flags being kept are the started and uniqueId.
All others will be set during the normal wallet process.